build.gradle

```diff
@@ -1,27 +1,10 @@
-subprojects { Project subproject ->
-    group "io.micronaut.nats"
-
-    if (subproject.name != 'docs-examples') {
-        apply plugin: "io.micronaut.build.internal.common"
-        apply plugin: "io.micronaut.build.internal.dependency-updates"
-    }
-
-    repositories {
-        maven { url "https://s01.oss.sonatype.org/content/repositories/snapshots/" }
-    }
-
-    if (subproject.parent.name == "docs-examples" || subproject.name == "docs-examples") {
-        return
-    }
-
-    apply plugin: "io.micronaut.build.internal.publishing"
+plugins {
+    id 'io.micronaut.build.internal.docs'
+    id 'io.micronaut.build.internal.dependency-updates'
+    id 'io.micronaut.build.internal.quality-reporting'
 }
 
-apply plugin: "io.micronaut.build.internal.docs"
-apply plugin: "io.micronaut.build.internal.dependency-updates"
-
-
-def prepareGuideHotfix = tasks.register("prepareGuideHotfix") {
+def prepareGuideHotfix = tasks.register('prepareGuideHotfix') {

```

Comment:
:o: Is this still required? The task looks very suspicious. I suggest to run `./gradlew clean`, then `./gradlew docs` and check the output.

Reply:
If removed the whole block ;)